Jenkinsfile

```
@@ -45,7 +45,7 @@ def sanitized_JOB_NAME = JOB_NAME.toLowerCase().replaceAll('/', '-').replaceAll(
 
 def el7_component_repos = ""
 def sle12_component_repos = ""
-def component_repos = "spdk fio dpdk pmdk raft"
+def component_repos = "spdk@PR-9 fio dpdk@PR-6 pmdk@PR-6 raft"
```

pmdk, fio and raft packaging updates have landed:

```suggestion
def component_repos = "spdk@PR-9 dpdk@PR-6"
```